src/activities/activities.txt

```diff
@@ -107,6 +107,7 @@ paintings
 penalty
 photo_hunter
 planegame
+rahulworld
```

Comment:
for the exercise, it's ok but for future, it would be nice to have a real name for the activity ;)